Add 'please add timestamps' command and tests

timrisi opened the pull request.

Commit:
Add 'please add timestamps' command and tests

gregoryjscott commented on app/Library/Timestamp/Tasks/GetFiles.cs:
```diff
@@ -0,0 +1,37 @@
+
+namespace Library.Timestamp.Tasks
+{
+    public class GetFiles : InOutTask<string, TimestampFile[]>
```
Please explicitly define the input and output with Input and Output classes. This will require naming the output and input properties which adds readability. https://github.com/gregoryjscott/Simpler/blob/docs/documentation.md#inouttask

gregoryjscott commented on README.md:
```diff
@@ -31,6 +31,15 @@ Command for running a batch of .sql files in a directory on a given database.
+Command for adding timestamps to all files in a directory that don't already have them.
+
+### Usage
+* 'please add timestamps in .\directory'
```
Need to use the ` character (to the left of 1 on the keyboard) instead of  the ' character to add the desired style on this line and the lines below.

gregoryjscott commented on app/Tests/Timestamp/Tasks/TimestampFileTest.cs:
```diff
@@ -0,0 +1,104 @@
+            All
+        }
+
+        public void refreshTestDirectory(TimestampState state)
```
The normal C# standard is Pascal case on all method names (although we break that rule on tests methods).

Commit:
Style Changes

Explicitly define input and output in GetFiles
Use the ` character instead of ' in the ReadMe
Change the non-test methods in TimestampTests to PascalCase

gregoryjscott committed:
add please test and remove invalid directory test cases

gregoryjscott committed:
rename task to verb

gregoryjscott committed:
drop the s
